my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/dao/EmergencyContactMiddlewareDaoImpl.java

```diff
@@ -48,7 +48,8 @@ public EmergencyContactMiddlewareDaoImpl(DataSource ds) {
       //return saved content
       return emergencyContacts;
     } else {
-      throw new Exception("There was an issue saving the emergency contacts");
+      throw new Exception("Error saving contacts for netid '" + netId + "'; "
+          + "Contacts that failed to save were " + emergencyContacts + ".");
```

Comment:
Are we concerned of the level of information we are dumping in a log? Since this data could be sensitive?  May want to setup a different log for this?